content/blogposts/2022/release-3.42

```diff
@@ -0,0 +1,55 @@
+---
+title: "Sourcegraph 3.42 release"
+publishDate: 2022-07-22T10:00-07:00
+description: "Sourcegraph 3.42 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.42"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+
+---
+
+Sourcegraph 3.42 is now available! For this release, we introduced:
+
+## All-new Admin Analytics 
+
+We believe providing a world-class admin-experience is critical to the future of Sourcegraph customers around the world. This starts with empowering administrators to measure and track the value of Sourcegraph within their organization.
+
+Historically, Sourcegraph administrators rely on word-of-mouth feedback from developers, leading to a poor understanding of the value of Sourcegraph within an organization. 
+
+In 3.42, we are releasing a completely redesigned user analytics experience for administrators. These visualizations will make it easier to understand developer engagement, and ultimately better manage user licenses, identify power users, and convey value to their internal leaders. 
+
+## Improved search-based navigation for Python non-locals
+We previously announced we’d [improved our search-based support for Python locals in Sourcegraph 3.40(https://about.sourcegraph.com/blog/release/3.40) by using tree-sitter. We've now extended this improvement for jump-to-definition to more kinds of variables such as class fields.
+
+## Beta UI improvements
```

Comment:
@benvenker Is there any more narrative you can provide around why we decided to re-tune the UI, or the design principles that are being used to make the changes?

From an external perspective (just checking on it myself), it feels like the filters column just moved to the right, so I think it would help to provide more explanation to the user audience on what's happening there

Reply:
@iskyOS the major improvements so far are:
- filters moved to the right and are collapsible now
- removed excessive buttons in the header of the search results page
- search results are more compact and show fewer matches by default; they're also individually expandable with sticky headers and a sticky "show fewer" footer that is always visible, even if the results expand below the fold

This is part of a long term effort to clean up the core workflow (code search) and make it more focused and these changes are the first steps.